openproblems/tasks/_cell_cell_communication/_common/methods/liana.py

```diff
@@ -67,70 +66,210 @@ def liana(
 
 
 @_liana_method(
-    method_name="CellPhoneDB",
+    method_name="RobustRankAggregate MAX",
```

Comment:
Do you want to call this `RobustRankAggregate` or simply `LIANA`?

Reply:
It's kind of both I guess. I will rename to LIANA's RRA (max). LIANA is not a method and this is just a 'fancy' way to aggregate the ranks of the methods in liana.

Reply:
or LIANA Rank Aggregate (max) is perhaps better :)

Reply:
Works for me!